lib/setup.rb

```diff
@@ -6,120 +6,69 @@
 
 
 class Setup
-  attr_reader :computer,
+  attr_reader :ships,
               :player,
-              :computer_board,
-              :player_board,
-              :ships
-
+              :computer
```

Comment:
Is this supposed to be an attr reader?

Reply:
I think the GitHub rendering makes it a bit confusing whats going on here. but i double checked the code, and it looks correct. this is what we have

<img width="265" alt="Screen Shot 2020-12-15 at 1 24 32 PM" src="https://user-images.githubusercontent.com/72056427/102268718-82973780-3ed0-11eb-9507-a4d00b9e24dc.png">

Reply:
Ah! Okay, thank you for the clarification here. Yes, paying attention to the line numbers on Git Hub is important. I appreciate the screen share just to confirm!